### PR TITLE
added assignment.rb and countries.txt

### PR DESCRIPTION
**Pull request by easmith2**
* Ran in circles trying to find a better way to do lines 20-32, and account for capitalization of words preceded by special symbols. Alas, all attempts at that failed; at least this accomplishes the basic goal.

### CODE REVIEW

**Comment by easmith2** [comment]
so.... if we push updates to the dev branch after creating a PR, do we need to update the PR at all? Or is this voodoo magic covering that for us?

File: assignment.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#Following mess is to undo some unwanted capitalization/spacing I caused.
+	country_name[i].gsub!(" And "," and ")
+	country_name[i].gsub!(" Of"," of")
+	country_name[i].gsub!(" The"," the")
+	country_name[i].gsub!(" U.s."," U.S.")
+	country_name[i].gsub!("( ","(")
+#Oh yea... gotta undo the undoing of some unwanted ... wat?
+	country_name[i].gsub!(", the",", The")
+#There must be a better way... I'll figure it out eventually (maybe)
```
Comment (rpearce) [inline assignment.rb]: Let's take some time tomorrow and figure this out! Nothing is impossible

File: assignment.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+	all_the_things[i] = country_abbrev[i] + " - " + country_name[i]
+	updated_data << all_the_things[i] + "\n"
+end
+
+puts "There are #{just_the_data.count} countries accounted for here.\nWould you like to see all of them? (y/n)"
```
Comment (rpearce) [inline assignment.rb]: this is awesome

**Comment by rpearce** [comment]
:+1:
If you'd like to continue to work on this, great. If not, close and merge this.

**Comment by rpearce** [comment]
Check this out:
https://github.com/andrewtpoe/1.2-Countries/pull/1/files

**Pull request merged**